packages/ckeditor5-link/src/utils/automaticdecorators.js

```diff
@@ -85,4 +87,42 @@ export default class AutomaticDecorators {
 			}, { priority: 'high' } );
 		};
 	}
+
+	/**
+	 * Provides the conversion helper used in the {@link module:engine/conversion/downcasthelpers~DowncastHelpers#add} method
+	 * when linking images.
+	 *
+	 * @returns {Function} A dispatcher function used as conversion helper
+	 * in {@link module:engine/conversion/downcasthelpers~DowncastHelpers#add}.
+	 */
+	getDispatcherForLinkedImage() {
```

Comment:
In most places (used as an independent util) it would be: `downcastLinkedImage()` or `getDowncastLinkedImageConverter()`.

The other thing is that I think that this is a responsibility of the `LinkImageEditing` - and maybe we could expose `this._definitions` somehow.

Reply:
OK... I see that you followed the pattern form here. So I've just left the comments above - don't need to change anything.

Let's keep that consistent.